src/main/java/de/muenchen/isi/domain/service/AbfrageStatusService.java

```diff
@@ -1,59 +1,274 @@
-                .getInfrastrukturabfrageById(id);
-        abfrage.setAbfrage(this.freigabeAbfrage(abfrage.getAbfrage(), id));
-        this.abfrageService.updateInfrastrukturabfrage(abfrage);
+    public void keineBearbeitungNoetig(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.KEINE_BEARBEITUNG_NOETIG, stateMachine);
+    }
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#IN_BEARBEITUNG_FACHREFERATE}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void verschickenDerStellungnahme(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.VERSCHICKEN_DER_STELLUNGNAHME, stateMachine);
+    }
 
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#BEDARFSMELDUNG_ERFOLGT}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void bedarfsmeldungErfolgt(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.BEDARFSMELDUNG_ERFOLGTE, stateMachine);
+    }
+
+    /**
+     * Ändert den Status auf {@link StatusAbfrage#ERLEDIGT}.
+     *
+     * @param id vom Typ {@link UUID} um die Abfrage zu finden.
+     * @throws EntityNotFoundException          falls die Abfrage nicht gefunden werden kann.
+     * @throws AbfrageStatusNotAllowedException wenn die Statusänderung nicht erlaubt ist
+     */
+    public void speichernVonSozialinfrastrukturVersorgung(final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        final StateMachine<StatusAbfrage, StatusAbfrageEvents> stateMachine = this.build(id);
+        this.sendEvent(id, StatusAbfrageEvents.SPEICHERN_VON_SOZIALINFRASTRUKTUR_VERSORGUNG, stateMachine);
+    }
+
+
+    /**
+     * Erstellt die Statemachine und initialisiert diese mit dem Status der Abfrage aus der DB.
+     * <p>
+     * Ausserdem wird ein preTransition-Listener definiert, der bei einer Statusänderung den Status in der DB aktualisiert.
+     *
+     * @param id vom Typ {@link UUID}  um die Abfrage aus der DB zu holen.
+     * @throws EntityNotFoundException falls die Abfrage nicht gefunden werden kann.
```

Comment:
Anmerkung um es einheitlich mit bereits bestehenden Javadocs zu machen.

```suggestion
     * @throws EntityNotFoundException falls die Abfrage nicht gefunden wird
```